### PR TITLE
Updated statinamic 0.8 -> 0.9

### PR DESCRIPTION
**Pull request by marcustisater**
@thangngoc89 - it says babel-register is missing. Hum...?

### CODE REVIEW

**Comment by thangngoc89** [comment]
@MoOx

File: package.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
     "stylelint-staged": "stylelint-staged",
     "svgo-staged": "./scripts/svgo.sh",
     "statinamic": "cross-env BABEL_DISABLE_CACHE=1 BABEL_ENV=statinamic DEBUG=statinamic:* babel-node scripts/build",
```
Comment (thangngoc89) [inline package.json]: You can remove this npm command now :)

File: package.json [hunk 2]
```diff
@@ -84,24 +70,18 @@
...
     "lint-staged": "^0.1.1",
-    "markdown-it": "^5.0.3",
-    "markdown-it-toc-and-anchor": "^1.0.1",
     "postcss-cssnext": "^2.4.0",
     "postcss-loader": "^0.7.0",
     "pre-commit": "^1.1.2",
-    "react": "^0.14.7",
-    "react-dom": "^0.14.7",
+    "react": "^15.0.0-rc.1",
+    "react-dom": "^15.0.0-rc.1",
```
Comment (thangngoc89) [inline package.json]: Are you sure ? React 15 RC is not stable.

**Comment by thangngoc89** [comment]
Also, Statinamic@0.9.0 requires `whatwg-fetch@0.11.0`

**Comment by thangngoc89** [comment]
> babel-register is missing 

You can work around it by using npm@3 :D

File: package.json (hunk 2 above)
Comment (MoOx) [inline package.json]: React master branch is constantly used in production at Facebook, so we cannot say that it's "not stable". :)

**Comment by marcustisater** [comment]
Sounds fair, let me know if you would you like me to add `whatwg-fetch@0.11.0` & `babel-register`. 

I thought babel-register was in core?

**Comment by MoOx** [comment]
Forget the babel-register thing for now, we pushed a fix as 0.9.2.
For fetch, no big deal but better to upgrade.

**Comment by marcustisater** [comment]
Alright, cool. I'll do some fixes after work hours.

**Comment by thangngoc89** [comment]
For installing npm@3 on node 4, you can see it at this line 
https://github.com/MoOx/statinamic/blob/2ab325c12d32b199e37488e1695313e35f9f1aea/.travis.yml#L22

peerDeps in npm@2 if frustrating.

**Comment by marcustisater** [comment]
Yup, let's go with npm@3

Thanks for the quick feedback.

**Comment by marcustisater** [comment]
Sorry, been caught up with a lot of work. I'll do the other ~~statinamic~~ Phenomic updates  in a new pull requests.

All good for merge @thangngoc89? 😄

File: .travis.yml [hunk 3]
```diff
@@ -3,6 +3,9 @@ node_js:
   - '5'
   - '4'
 
+before_install:
+  - npm install -g npm@latest
+
```
Comment (MoOx) [inline .travis.yml]: Not required anymore.  At least for node 5.  By the way I think you should disable  node 4 on Travis.

Comment (marcustisater) [inline .travis.yml]: Ok, just roll with npm 5? Thanks for quick input.

Comment (MoOx) [inline .travis.yml]: I think in your case, node 5 is enough.

**Comment by MoOx** [comment]
Lgtm. @thangngoc89  won't be  available for a few days.

**Comment by marcustisater** [comment]
Alright, good to know. Thanks

**Comment by MoOx** [comment]
:+1:

**Pull request merged**